dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionOrganization.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  *    Copyright 2019 OICR
+ *  *
+ *  *    Licensed under the Apache License, Version 2.0 (the "License");
+ *  *    you may not use this file except in compliance with the License.
+ *  *    You may obtain a copy of the License at
+ *  *
+ *  *        http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  *    Unless required by applicable law or agreed to in writing, software
+ *  *    distributed under the License is distributed on an "AS IS" BASIS,
+ *  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  *    See the License for the specific language governing permissions and
+ *  *    limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.core;
+
+import org.hibernate.Hibernate;
+
+/**
+ * This class is used as the response object of the entry collections endpoint.
+ * @author gluu
+ * @since 1.6.0
+ */
+public class CollectionOrganization {
+    public final Collection collection;
+    public final Organization organization;
+
+    public CollectionOrganization(Collection collection, Organization organization)  {
+        Hibernate.initialize(organization);
```

Comment:
This seems heavy-handed, how large does the response get?
And what information are you trying to get?

Reply:
More importantly, because some data doesn't get serialized. How many hibernate SQL queries would this kick off?

Reply:
Same amount as /starredUsers endpoint...probably not a great thing but bearable?